draft-wang-ppm-dap-taskprov.md

```diff
@@ -192,6 +192,10 @@ struct {
 } QueryConfig;
 ~~~
 
+In particular, if `query_type` is `fixed_size` and `max_batch_size` is 0,
```

Comment:
Grammar: "In particular" is used to refine a previous statement, whereas here it's beginning a statement. I think what you want to do instead is start the paragraph off with "The max batch size for the `fixed_size` query is optional."